GameSalesApi/GameSalesApi/Features/AccountManagement/Commands/UpdateUserCommand.cs

```diff
@@ -21,6 +21,16 @@ public class UpdateUserCommand : ICommand<Result>
         /// </summary>
         public string Email { get; set; }
 
+        /// <summary>
+        /// New username
```

Comment:
New?

Reply:
We can't change username or what question?